src/App.tsx

```diff
@@ -1,109 +1,37 @@
 import React from 'react';
-
 import './App.scss';
-
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { Post } from './types/Post';
+import { User } from './types/User';
+import { Comment } from './types/Comment';
+import { PostList } from './components/PostList';
+
+function getUserById(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+function filterCommentsById(postId: number): Comment[] {
+  const foundComment = commentsFromServer.filter(
```

Comment:
Indentation should be consistent throughout the code. There is an extra space before the 'const' keyword. Consider updating the indentation to match the rest of the code.

Reply:
There is only one tab, there is no extra space there.